src/extension/tests/helpers/1234.settings

```diff
@@ -11,7 +11,8 @@
                 "rebootSetting": "IfRequired",
                 "classificationsToInclude":["Critical","Security"],
                 "patchesToInclude": ["*", "test*","*ern*=1.2*", "kern*=1.23.45"],
-                "patchesToExclude": ["*", "test", "*test"]
+                "patchesToExclude": ["*", "test", "*test"],
+                "maxPatchPublishDate": "2019-07-20T12:12:14Z"
```

Comment:
Same comment about bifurcation of tests -- for all new tests for the feature, use a different settings file.

Reply:
Same answer as above :)